metrics/examples/prometheus/kubernetes.yaml

```diff
@@ -436,6 +439,33 @@ data:
         replacement: $1
         action: replace
 
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: prometheus-rules
+data:
+  rules.yml: |
+    groups:
+    - name: kafka
+      rules:
+      - alert: UnderReplicatedPartitions
+        expr: kafka_server_replicamanager_underreplicatedpartitions > 0
+        for: 10s
+        labels:
+          severity: warning
+        annotations:
+          summary: 'Kakfa under replicated partitions'
+          description: 'There are {{ $value }} under replicated partitions on {{ $labels.kubernetes_pod_name }}'
+      - alert: NoActiveController
+        expr: sum(kafka_controller_kafkacontroller_activecontrollercount) == 0
+        for: 10s
+        labels:
+          severity: warning
+        annotations:
+          summary: 'Kakfa no active controller'
+          description: 'There is no active controller in the cluster'
```

Comment:
I know this is "only an example", but I think we should try to be a bit more opinionated about the conditions people should be alerting about. Could we add:

* \# of under minIsr partitions
* \# of offline log directories

Reply:
Well, there are "a lot" of other useful metrics that we could add (i.e. offline partitions count, isr shrink/expand per sec, ...)  so it's difficult to be opinionated here or we should add a bunch of them.

Reply:
Most of the exposed metrics aren't suitable for alerting though. I don't think isr shrink/expand per sec is, for example. offline partitions count might be worth adding though, as well as the two I mentioned. https://blog.serverdensity.com/how-to-monitor-kafka/ has a few more ideas. I'm not saying we should do all of them, but we should consider all of them.